crimson/utils.py

```diff
@@ -43,6 +43,8 @@ def convert(raw_str):
     maybe_float = RE_FLOAT.search(raw_str)
     if maybe_float is not None:
         return float(maybe_float.group(1))
+    if raw_str == 'NA':
```

Comment:
Would it be possible to tweak this check such that we can check for multiple variants of `NA`s? Something like adding `str_none_values=(...)` as one of the function's argument seems ok. The check would then become `if raw_str in str_none_values`.

That way, we can have the function accept `NA`, `na`, `n/a`, `N/A`, and/or any other variant we want to set as None values.